anta/tests/interfaces.py

```diff
@@ -474,8 +502,8 @@ class Input(AntaTest.Input):
 
         mtu: int = 1500
         """Default MTU we should have configured on all non-excluded interfaces. Defaults to 1500."""
-        ignored_interfaces: list[str] = Field(default=["Management", "Loopback", "Vxlan", "Tunnel"])
-        """A list of L3 interfaces to ignore"""
+        ignored_interfaces: list[AllInterfacePrefix | Interface] = Field(default=["Management", "Loopback", "Vxlan", "Tunnel"])
+        """A list of L3 interfaces to ignore."""
```

Comment:
```suggestion
        """A list of L3 interfaces or interface types to ignore."""
```
If we click on `InterfaceType` or `Interface`, check if we have an example or if it's clear enough what we expect here in the documentation. If not, we should probably add something in the custom types or here.

Reply:
We could also add more details in the example docstring above.

Reply:
Updated docstrings. Thanks.